tedana/workflows/tedana.py

```diff
@@ -649,6 +649,21 @@ def tedana_workflow(
             io_generator.save_file(utils.millisec2sec(t2s_limited), "limited t2star img")
             io_generator.save_file(s0_limited, "limited s0 img")
 
+    rmse_map, rmse_timeseries, rmse_sd_timeseries = decay.model_fit_decay_ts(
+        data=catd,
+        tes=tes,
+        adaptive_mask=masksum_denoise,
+        t2s=t2s_limited,
+        s0=s0_limited,
```

Comment:
That's a good point. I actually think it would be useful evaluate model fit when you provide your own T2* map, but that would require adding a step that estimates S0 from the data and T2*, which is just too much for this PR.

I think we'll need more advanced templating, like Jinja2, to handle missing figures. BTW I think that's already an issue with things like the T2* maps when rerunning tedana, so it's worth opening an issue about.